pnpm-lock.yaml

```diff
@@ -26,15 +25,18 @@ specifiers:
   husky: ^7.0.2
   is-ci: ^3.0.0
   nuxt: ^2.14.6
+  nuxt-fontawesome: ^0.4.0
```

Comment:
you dont have to use `nuxt-fontawesome` unplugin-icons already has them all so use it instead